src/pages/Question.js

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react'
+import Logo from '../components/Logo.js'
+import Header from '../components/Header.js'
+import  {questions} from '../config';
+import Answer from '../components/Answer.js'
+import '../styles/questions.scss';
+import PreviousButton from '../components/PreviousButton.js';
+
+const Question = () => {
+    const categories = ['food', 'sleep', 'sport', 'misc'];
+    const [currentCategory, setCurrentCategory] = useState(0)
+    const [currentQuestion, setCurrentQuestion] = useState(1)
+
+    let categoryObject = questions[categories[currentCategory]]
+    let questionObject = categoryObject.questions[currentQuestion]
+
+    const nextQuestion = () => {
+
+        setCurrentQuestion(currentQuestion + 1)
+        if (!((currentQuestion+1) in categoryObject.questions)) {
```

Comment:
What does this mean? Statements like this can be put in variables. This way your code will be more descriptive and understandable. I think your checking if you're at the last question or not, maybe put this statement in a var isLastQuestion or something like that